lib/matplotlib/axes/_axes.py

```diff
@@ -834,6 +834,75 @@ def axvline(self, x=0, ymin=0, ymax=1, **kwargs):
         self.autoscale_view(scalex=scalex, scaley=False)
         return l
 
+    @docstring.dedent_interpd
+    def axline(self, xy1, xy2, **kwargs):
+        """
+        Add an infinitely long straight line that passes through two points.
+
+        Parameters
+        ----------
+        xy1, xy2 : length 2 iterables
```

Comment:
Does someone know the correct way to say `length 2 iterables`?

Reply:
I would just say `two pairs of floats` (which is a bit too strict but good enough in practice IMO...).

Reply:
When discussing types, we came up with

> Use (float, float) to describe a 2D position.

https://matplotlib.org/devdocs/devel/documenting_mpl.html#parameter-type-descriptions